src/Psa/Printer.purs

```diff
@@ -53,12 +55,12 @@ instance monoidRow :: Monoid (Row a) where
 
 data AnsiText
   = Plain String
-  | Style (Array Ansi.GraphicsParam) String
+  | Style (NonEmptyList Ansi.GraphicsParam) String
```

Comment:
What was the reason for the `NonEmptyList` change? Also why not `NonEmptyArray`?

Reply:
I think purescript-ansi was changed? I was also a bit confused, but I just rolled with it.